runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -247,6 +247,17 @@ public void onConfigChanged(ConfigChanged event)
 			}
 		}
 
+		if (event.getKey().equals("isResizable"))
+		{
+			SwingUtilities.invokeLater(() ->
+			{
+				Dimension size = client.getSize();
+				this.setResizable(Boolean.valueOf(event.getNewValue()));
+				client.setSize(size);
```

Comment:
What is the point of setting the size here if resizable is toggled on/off? Isn't ```size``` already the size of the client?

Reply:
When you change the resizability option the drag area appears/disappears and the window changes size a little bit. Before I put that in, the window would grow or shrink when you toggled the option.